src/components/LoginSignUp/AuthenticationService.js

```diff
@@ -215,14 +223,18 @@ const verifyCredentials = async (
   formData.append( "username", username );
 
   const api = createAPI();
-  let response = await api.post( "/oauth/token", formData );
+  let response = await api.post( "/oauth/token", { locale: i18next.language }, formData );
```

Comment:
According to the [apisauce docs](https://github.com/infinitered/apisauce#calling-the-api), this just replaces the entire POST body with `{ locale: i18next.language }` and passes `formData` as a config object to axios. Instead I think you need to put the `locale` in the `formData`.